src/Host/Client/Impl/BrokerServices/RemoteUriWebService.cs

```diff
@@ -14,22 +14,20 @@
 using Microsoft.AspNetCore.WebSockets.Protocol;
 using Microsoft.Common.Core;
 using Microsoft.Common.Core.Logging;
-using Microsoft.Common.Core.Services;
+using Microsoft.Common.Core.Shell;
 using Microsoft.R.Host.Protocol;
 using static System.FormattableString;
 
 namespace Microsoft.R.Host.Client.BrokerServices {
     public class RemoteUriWebService : IRemoteUriWebService {
-        private readonly ICoreServices _services;
+        private readonly ICoreShell _coreShell;
         private readonly IConsole _console;
-        private readonly IActionLog _log;
 
-        public RemoteUriWebService(string baseUri, ICoreServices services, IConsole console) {
+        public RemoteUriWebService(string baseUri, ICoreShell coreShell, IConsole console) {
```

Comment:
It is more for convenience since shell has `Log()` extension. I can add container extensions too though.